src/styles/main.scss

```
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+@for $i from 1 through 31 {
+  .calendar__date-value:nth-child(#{$i}):before {
+    content: "#{$i}";
+    font-size: 30px;
+    font-family: "Arial", sans-serif;
+  }
+}
+
+$days: (
+  mon: "Monday",
+  tue: "Tuersday",
+  wed: "Wensday",
+  thu: "Thursday",
+  fri: "Friday",
+  sat: "Saterday",
+  sun: "Sunday"
+);
+
+.value {
+  width: 100px;
+  margin-right: 10px;
+}
+
+@each $name, $day in $days {
+  .calendar__day-#{$name}::before {
+    content: "#{$day}";
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    color: red;
+  }
+}
+
+.calendar__day {
+  display: flex;
+}
+
+@for $i from 28 through 31 {
+  .month-length:nth-child(#{$i}):before {
+    content: " ";
+  }
+}
+
+.calendar__date {
+  display: grid;
+  grid-template-columns: repeat(7, 0fr);
+}
+
+.calendar__date-value {
+  background-color: #eee;
+  width: 100px;
+  height: 100px;
+  border: 1px solid black;
+  box-sizing: border-box;
+  margin-bottom: 1px;
+  margin-right: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 1px 10px;
+  transition-duration: 0.5s;
```

don't use transition-duration, bc it's too hard for the browser
use transition: transform $transitionDuration, color ....
and add value for that you need transition duration
also, create a variable for this value
```suggestion
  transition: ... $trasitionDuration;
```